Vagrantfile

```diff
@@ -0,0 +1,35 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+hostname      = 'st2-packages'
+st2user       = ENV['ST2USER'] ? ENV['ST2USER']: 'st2admin'
+st2passwd     = ENV['ST2PASSWORD'] ? ENV['ST2PASSWORD'] : 'Ch@ngeMe'
+box           = ENV['BOX'] ? ENV['BOX'] : 'centos/7'
```

Comment:
See this approach: https://github.com/StackStorm/ansible-st2/blob/master/Vagrantfile#L5-L22